nmdc_runtime/site/resources.py

```diff
@@ -194,15 +194,17 @@ def get_object_bytes(self, object_id) -> requests.Response:
             access = AccessURL(
                 **self.get_object_access(object_id, method.access_id).json()
             )
-            if access.url.startswith(
+            if str(access.url).startswith(
```

Comment:
Side note: Maybe there's already a way to get the `url` property of `access` as a string, rather than casing it here. I noticed the `AccessURL` class has this method defined (I don't know what the decorator does yet).

https://github.com/microbiomedata/nmdc-runtime/blob/7bc8f4066898a73863700b12b3d1e66abf31e5bc/nmdc_runtime/api/models/object.py#L31-L37

Reply:
the `field_serializer` decorator ensures that when `.model_dump()` is called on a model, `url` is stringified, but this doesn't hold for direct attribute access, i.e. `<model>.url`.